extensions/nature-sounds/src/Actions.tsx

```diff
@@ -0,0 +1,59 @@
+import { ActionPanel, Action, Toast, showToast } from "@raycast/api";
+import { Audio, Item, PlayingItem } from "./types";
+import { exec } from "child_process";
+
+export default function Actions(props: { item: Audio; playing: Item[]; setPlaying: PlayingItem["setPlaying"] }) {
+  return (
+    <ActionPanel title={props.item.name}>
+      <ActionPanel.Section>
+        {props.item.filename === "stop" && (
+          <Action
+            title="Stop All Sounds"
+            onAction={() => {
+              exec(`killall afplay`);
```

Comment:
It could be better to loop the array of playing items and killing them individually since we don't know if the user plays anything else with `afplay`

Reply:
```suggestion
              props.playing.forEach((item) => {
                exec(`kill -9 ${item.pid + 1}`);
              });
```
And this could work here